markdownify/__init__.py

```diff
@@ -89,6 +90,34 @@ def should_remove_whitespace_outside(el):
     return should_remove_whitespace_inside(el) or (el and el.name == 'pre')
 
 
+def _is_content_element(el):
+    """Returns True for content (tags and non-whitespace text), else False."""
```

Comment:
Maybe include an example of a non-content element in the comment

Reply:
Good feedback! Implemented.